modules/standard/Sys.chpl

```diff
@@ -246,6 +246,16 @@ module Sys {
   extern proc sys_strerror(error:err_t, ref string_out:c_string):err_t;
 
   extern proc sys_readlink(path:c_string, ref string_out:c_string):err_t;
+
+  /*Check if the environment variable `name` is defined or not.
+    If `name` is defined then return 1 and update `string_out` 
+    as the value of environment variable
+    else the function returns 0.
+
+    :arg name: name of environment variable
+    :arg c_string: store the value of `name` environment variable if defined
```

Comment:
This second arg should be `string_out` (the argument name) rather than `c_string` (its type).  You can use additional lines to indicate the argument types (and return type) as well.  See https://chapel-lang.org/docs/modules/standard/Path.html#Path.realPath for an example and https://github.com/chapel-lang/chapel/blob/master/modules/standard/Path.chpl#L595-L606 for the comment string that generates it (noting that `realPath` also throws, where this function does not.

Note that in chpldoc / sphinx, single backtics like

     `name`

result in italicized names and that double backtics like

      ``name``

are required to get code-style formatting.  (I'm not sure I object to the italicized format per se, but often get this wrong due to all the markdown I write...  the few routines I happened to look at use the code-style formatting when referring to arguments in sentences, but I suspect I could find italic cases as well with more searching.

Reply:
> This second arg should be string_out (the argument name) rather than c_string (its type).

Oh! my bad.